test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestConfigPersistenceHelper.cs

```diff
@@ -0,0 +1,157 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.SqlTools.ServiceLayer.Test.Common
+{
+    public sealed class TestConfigPersistenceHelper
+    {
+        private static string DefaultSettingFileName = Path.Combine(FileUtils.UserRootFolder, "sqlConnectionSettings.json");
+        private static TestCredentialService _credentialService = TestCredentialService.Instance;
```

Comment:
Minor: please remove `_` at start of fields. That's a DacFx/SSDT naming pattern that we didn't bring over.